README.md

```diff
@@ -1,59 +1,104 @@
-README
-------
+OIP
+---
 
-Use libpcap and SDL to visualize IP traffic between multiple endpoints
+OrganicIP (OIP) visualizer uses libpcap and Simple DirectMedia Layer (SDL) to visualize IP traffic between endpoints.
 
-Packages
---------
- * `deb http://mirror.usu.edu/usu-security/ [codename]-security-unstable main`
-     * Debian: sid, bullseye
-     * Ubuntu: impish, focal, bionic
- * install `usu-archive-keyring` package and apt-get update
-     * keyring file available at `https://raw.githubusercontent.com/USU-Security/oip/master/usu_archive_keyring.gpg`
+Supported Distributions
+-----------------------
+
+The following operating systems are supported: 
+
+- Debian
+   - sid
+   - bullseye
+- Ubuntu 
+   - impish
+   - focal
+   - bionic
+
+Distribution
+------------
+
+Mirrors are available at following location:
+
+http://mirror.usu.edu/usu-security/dists/
+
+The repository keyring is located at the following location: 
+
+https://raw.githubusercontent.com/USU-Security/oip/master/usu_archive_keyring.gpg
 
 Colors
 ------
+
 On packet-balls:
  - Red for UDP
  - Green for TCP
  - White/blue for any other IP protocol
+
 On hosts:
  - defined by oip.conf (ABGR, with A ignored)
 
-
 Running
 -------
+
 Steps to run the server/client for a live capture
+
  - create /etc/oip.conf and at a minimum set a secret
  - start the server: `oipd <interface>`
  - start the client: `oip -f <pcap_filter> -s <server>`
 
 The client also supports reading from a pcap file
+
  - `oip -c <pcap_file>`, can use `-e <number>` to change speed (1000 is default, number of ms per 1 second in the pcap)
 
 Controls
 --------
+
 On the gui, you can use the mouse and the keyboard.
 
 Mouse controls:
+
  - left click on host to reposition
  - right click on host to bring up context menu
  - add actions to context menu in oip.conf on client
 
 Keyboard keys:
+
  - ! to open the menu
- - , to start displaying the capture (Load button).
+ - , to start displaying the capture (Load button)
  - p to pause and play the capture
- - <ESC> to disconnnect?
+ - <ESC> to disconnect
+
+Installation from Repository
+----------------------------
+
+To install OIP from existing repositories execute the following commands for Ubuntu-based : 
+
+```
+curl -L https://raw.githubusercontent.com/USU-Security/oip/master/usu_archive_keyring.gpg | sudo apt-key add -
+sudo add-apt-repository 'deb [arch=amd64] http://mirror.usu.edu/usu-security/ focal-security-unstable main'
+sudo apt-get install oip
+curl -L https://raw.githubusercontent.com/USU-Security/oip/master/oip.conf-example --output /etc/oip.conf
```

Comment:
It may be better to copy the example config from /usr/share/doc/oip/ in case of incompatible changes in the master branch

Reply:
Fair point! Went ahead and pushed updated syntax based off your suggestion.